surrealdb/clients/asynchronous.py

```diff
@@ -0,0 +1,168 @@
+"""
+Copyright © SurrealDB Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from json import JSONDecodeError
+from types import TracebackType
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Type
+
+import httpx
+
+from ..common import json as jsonlib
+from ..exceptions.surreal_exception import SurrealException
+from ..models.response import SurrealResponse
+
+__all__ = ("AsyncSurrealDBClient",)
+
+
+class AsyncSurrealDBClient:
```

Comment:
imo this class name is too long, is there anything shorter it could be?

Reply:
i don't see any good way to shorten it, i think it's fine until we get rid of the sync one (**if** we do)